independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/project/update/rewrite/QuarkusUpdateCommand.java

```diff
@@ -195,10 +195,23 @@ private static void executeRewrite(Path baseDir, Map<String, List<String>> comma
 
         log.info("");
 
-        String logInfo = logFile != null ? "Logs can be found at: %s".formatted(baseDir.relativize(logFile).toString())
-                : "See the execution logs above for more details";
+        String logInfo;
+
+        if (logFile != null) {
+            logInfo = "Logs can be found at: %s".formatted(baseDir.relativize(logFile).toString());
+
+            try {
+                // we delete the log file prior to executing the command so that we start clean
+                Files.deleteIfExists(logFile);
```

Comment:
I don't understand this part, when you say _Logs can be found at_ and than you delete the log file, you can't see it there. I presume that `baseDir.relativize(logFile)` refers to different file than `logFile`?

Reply:
execution happens on line 217 and `logInfo` is passed to the called method .. that's how I interpreted the code and was OK with it :)

Reply:
so you basically say that the loggin message is for the next file that will be generated? so why is it hidden behind `logFile != null`, it doesn't make sense to me

Reply:
okay, I understand, so when `logFile != null`, it wil be stored in the file that they create; makes sense

Reply:
@michalvavrik sorry for the late answer. It's a bit confusing but we open the file with an append or create later so that all the executed commands accumulate in the log file. So basically I delete it, so that we start fresh every time we execute an update.

Otherwise you end up with multiple iterations.

Pattern is so so but is probably good enough.